Suggested changes to Unacceptable Behavior section

danmactough opened the pull request:
- Removed language that ventured into legal territory
- Tried to clarify and improve some explanations

danmactough committed:
Suggested changes to Unacceptable Behavior section

- Removed language that ventured into legal territory
- Tried to clarify and improve some explanations

cpolitano commented:
:+1:

kylehill commented:
:+1:

gfogle commented on README.md:
```diff
@@ -36,11 +36,18 @@ Every member of the product team is expected to work hard, be considerate of the
+- never respond with surprise when someone asks for help;
+- and take care neither to patronize your colleagues nor assume complete knowledge of a topic.
+
+This last point is especially important when talking about technical topics: Women and people of color in the tech industry have many tales of being either [mansplained](https://www.guernicamag.com/daily/rebecca-solnit-men-explain-things-to-me/) about a field in which they are experts, or else excluded from learning opportunities because a colleague wouldn’t make an effort to answer questions. Do ...
```
while i realize using the term mansplained is meant to convey patronization and prejudice towards women / minorities, removing its reference and using more gender neutral language would promote intolerance of mistreatment in general. which i think is more in line with the ultimate goal of this document?

danmactough replied:
I agree that "mansplaining" is pejorative, which seems at odds with the CoC itself. To me, the term's usefulness outweighs that negative -- I can't think of an alternative term that captures this specific breed of boorish behavior that is so pervasive in a community dominated by men (white men, specifically). It's a really information-packed word that vividly evokes exactly the meaning it's trying to convey. (Although maybe that's because I've already made that association. Maybe to someone hearing the term for the first time, the pejorative poke is the only effect.)

I think it's more helpful to identify the behavior than to protect the feelings of the privileged, but if we can have **both** I'm all ears. What's the alternative?

gfogle replied:
`This last point is especially important when talking about technical topics: Engineers have many tales of being patronized about a field in which they are experts, or else excluded from learning opportunities because a colleague wouldn’t make an effort to answer questions. Don’t be that person. Remember that your colleagues may have expertise you are unaware of, and listen at least as much as you speak.`

how about that? i just think its more important to convey problems with the message delivery than the messenger

cpolitano replied:
@gfogle you make an important point here. Yes, as @danmactough articulates, the term "mansplaining" is both imbued with meaning but also pejorative. I agree that we can do better, and offer a clearer definition of the behavior we want to avoid. "Mansplaining" does violate the CoC, as both of you allude, because it suggests that only men exhibit this behavior. Meanwhile the core of intersectionality is the acknowledgement that members of oppressed groups can themselves be oppressive, and everyone has a responsibility to avoid these behaviors and strive to treat everyone else with the utmost respect. We're all in this together. 

That said, while Vox's CoC has been such a valuable jumping off point, I think we should scrap this paragraph entirely. Instead, how about adding a few more items to the list? Mainly:

```
- don't presume to explain a topic without knowing whether someone wants or needs an explanation
- strive to be inclusive in learning opportunities and technical conversations
- listen more than you speak
```

gfogle replied:
:+1:

danmactough replied:
@cpolitano I like that a lot. :100: Sure, losing the convenience of a label for a category of behavior is  :-1:, but the "mansplaining" label is really clunky (at best).

necaris replied:
:+1: to being explicit about the behaviors we want :+1:

ecgeiser commented:
:+1:

danmactough committed:
Replace pejorative term with more explicit language

re: https://github.com/socialtables/code-of-conduct/pull/9#discussion-diff-49153612

danmactough commented:
@cpolitano Updated to incorporate your language. 49b537b I think the "merge" button is yours.
